security/json_login_setup.rst

```diff
@@ -0,0 +1,229 @@
+How to Build a JSON Authentication Endpoint
+===========================================
+
+.. tip::
+
+    If you are storing users in some sort of a database, then you should consider
```

Comment:
Is it really relevant here ? IMHO it does not have specifically much to do with the current article :/